volatildap/server.py

```diff
@@ -144,8 +144,8 @@ def quote(base, *args):
             yield quote('TLSCertificateFile %s', self._tls_certificate_path)
             yield quote('TLSCertificateKeyFile %s', self._tls_key_path)
 
-        yield quote('moduleload back_hdb')
-        yield quote('database hdb')
+        yield quote('moduleload back_mdb')
```

Comment:
Just wondering whether `mapsize` should also be set when using this backend. The default is capacity is 10 MiB ([source](https://git.openldap.org/openldap/openldap/-/blob/0ed99989630064a89526c20dfeee069e26111493/servers/slapd/back-mdb/back-mdb.h#L49)).